anvil-rstudio-bioconductor/scripts/set_up_package_dir.sh

```diff
@@ -6,9 +6,10 @@ set -e
 # Therefore, we will run this script from Leonardo after the PD is mounted
 
 RSTUDIO_USER_HOME=/home/rstudio
-R_PATH=`R CMD printenv R_LIBS_USER | sed 's/~\///g'`
+R_PATH=`Rscript -e "cat(Sys.getenv('R_LIBS_USER'))"`
+# R_PATH=`R CMD printenv R_LIBS_USER | sed 's/~\///g'`
 BIOCONDUCTOR_VERSION=`printenv BIOCONDUCTOR_DOCKER_VERSION | sed 's/\(^[0-9].[0-9][0-9]\).*/\1/g'`
-R_PACKAGE_DIR=${RSTUDIO_USER_HOME}/${R_PATH}-${BIOCONDUCTOR_VERSION}
+R_PACKAGE_DIR=${RSTUDIO_USER_HOME}${R_PATH}-${BIOCONDUCTOR_VERSION}
```

Comment:
talked about this via slack but posting here for visibility since i'm out the rest of this week

the issue is that `R_PATH` is `/home/rstudio/R/x86_64-pc-linux-gnu-library/4.2` and `RSTUDIO_USER_HOME` is `/home/rstudio` so that would mean that `R_PACKAGE_DIR=${RSTUDIO_USER_HOME}/${R_PATH}-` is `/home/rstudio//home/rstudio/R/x86_64-pc-linux-gnu-library/4.2-` when it should just be `home/rstudio/R/x86_64-pc-linux-gnu-library/4.2-`

`R_PATH` used to be `R/x86_64-pc-linux-gnu-library/4.2` so we should make sure it's still getting that value

Reply:
R_PATH is actually `/root/R/x86_64-pc-linux-gnu-library/4.2`. 

The fix I issued now makes R_PATH `R/x86_64-pc-linux-gnu-library/4.2`.